src/chip/Chip.tsx

```diff
@@ -2,14 +2,47 @@ import React, { VFC } from 'react'
 import MuiChip, { ChipProps as MuiChipProps } from '@mui/material/Chip'
 import tw, { css, TwStyle, theme } from 'twin.macro'
 
+type ColorType = 'default' | 'negative'
+
 export type ChipProps = Exclude<MuiChipProps, 'color'> & {
   twin?: TwStyle[]
-  color?: 'default' | 'negative'
+  color?: ColorType
 }
 
-const styles: TwStyle = {
-  default: tw`bg-shade-light-default`,
-  negative: tw`bg-negative-light-default`,
+function styles(color: ColorType) {
```

Comment:
```suggestion
const styles = (color: ColorType) => {
```

他と合わせるならこうですかね？なにか理由があるのかもなのでおまかせします！